docs/messaging/kafka-operators-getting-started.md

```diff
@@ -47,22 +47,23 @@ host.name=myhost.mycompany.com
 
    * *Specify the -t or --spl-path command parameter when you run the sc command.*
 
-     `sc -t $STREAMS_INSTALL/toolkits/com.ibm.streamsx.kafka -M MyMain`
+     `sc -t \<kafka-toolkit-location\>/com.ibm.streamsx.kafka -M MyMain`
 
-   * *If  Streams Studio is used to compile and run SPL application, add Kafka toolkit to toolkit locations in Streams Explorer by following [these directions](http://www.ibm.com/support/knowledgecenter/SSCRJU_4.1.1/com.ibm.streams.studio.doc/doc/tusing-working-with-toolkits-adding-toolkit-locations.html?lang=en).*
+   * *If  Streams Studio is used to compile and run SPL application, add Kafka toolkit to toolkit locations in Streams Explorer by following [these directions](https://www.ibm.com/support/knowledgecenter/SSCRJU_4.3.0/com.ibm.streams.studio.doc/doc/tusing-working-with-toolkits-adding-toolkit-locations.html).*
 
```

Comment:
Could you add a comment for using VS Code... and adding a toolkit:
https://ibmstreams.github.io/streamsx.documentation/docs/spl/quick-start/qs-3/#how-do-i-add-a-toolkit-as-a-dependency

Reply:
commit 17bdd4f0442d665de47f14d8abd06ec11a9db360 added.